base/v0_5_exp/translate.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	"github.com/coreos/go-systemd/v22/unit"
 	"github.com/coreos/ignition/v2/config/util"
-	"github.com/coreos/ignition/v2/config/v3_4_experimental/types"
```

Comment:
For legibility reasons, revendoring is an exception to the rule that every commit should be an independent change.  Let's split out the revendoring, even though the resulting commit won't build.